lightning/src/routing/router.rs

```diff
@@ -333,79 +429,199 @@ pub fn get_route<L: Deref>(our_node_id: &PublicKey, network: &NetworkGraph, targ
 		};
 	}
 
-	match network.get_nodes().get(target) {
-		None => {},
-		Some(node) => {
-			add_entries_to_cheapest_to_target_node!(node, target, 0);
-		},
-	}
+	let mut payment_paths = Vec::new();
+	let mut tries_left = 10;
+	let mut found_amount_msat = 0;
+
+	// TODO: currently pretending that always have blockchain/amount data for channel capacities.
```

Comment:
Right...not sure what the "right" answer is. Maybe treat channels with unknown value as some common but small value (0.01 or 0.005 BTC?) and hope people upgrade to include the channel value in the announcement?

Reply:
Hmm, do we have numbers for how common they are on the network today? I think I agree, but should at least sanity-check.

Reply:
I thought the main scenario of not-knowing channel capacities is when the node doesn't have access to UTXO?

Reply:
The receiving node not having access to the chain is likely a common case, the question for such nodes is how many `channel_update` messages floating around the network don't have `htlc_maximum_msat` provided to compensate for the lack of chain access.